edenia/antelope-tools
# Feat/non compliant bps 940

### Add section with the list of the non-compliant Block Producers

### What does this PR do?

- Resolve #940
- Add the new section
- Add the route in the sitemaps
- New design in cards of Rewards Distribution
- Refactor Rewards Distribution state
- Remove more info icon in BPs section

### Steps to test

1. Run the project locally, preferably in eos mainnet
2. Go to "/non-compliant-bps" 
3. Check the content

### Screenshots

# Non-compliant

## Desktop

![imagen](https://user-images.githubusercontent.com/66583677/213483768-abf529b7-a661-41c1-848f-a4f8e3e9dee9.png)
![imagen](https://user-images.githubusercontent.com/66583677/213483845-8a0ea96d-7a06-4ec2-9a71-fdf49b65e760.png)

## Mobile

![imagen](https://user-images.githubusercontent.com/66583677/213484060-1e673b1d-e073-4523-a2db-52d5ccc4ece8.png)
![imagen](https://user-images.githubusercontent.com/66583677/213484111-a7cefff0-8205-42cd-a372-8b681adbae09.png)

# Rewards Distribution

## Desktop

![imagen](https://user-images.githubusercontent.com/66583677/213484180-c146a705-9510-440e-8f3d-03ce1917b9fc.png)
![imagen](https://user-images.githubusercontent.com/66583677/213484220-cbda0b09-1b9a-4cff-9c79-74de3f28c23d.png)

## Mobile

![imagen](https://user-images.githubusercontent.com/66583677/213484512-ada83db3-5448-4a94-8f24-1c738064f93b.png)

# Block Producers

![imagen](https://user-images.githubusercontent.com/66583677/213484795-6477bebf-8292-4a6f-8337-d1d23de21581.png)


#### CheckList
